src/ts/Offer/Create/Inputs.tsx

```diff
@@ -30,6 +30,7 @@ const inputGenerator = (type, key, label) => {
       return (
         <div className="col">
           <FormsyText
+            id={`offer-create-${label}`}
```

Comment:
Yes, `material ui` generates own `id`'s that's why we had a problem with server side rendering, I described this in issue #57